MultiTable/src/main.js

```diff
@@ -248,7 +248,7 @@ function setSampleListeners(subjectGroup) {
       }
 
       if (s.relatedLinks && s.relatedLinks.length > 1) {
-        s.relatedLinks.sort(Utils.sort);
+        s.relatedLinks.sort(Utils.sortByNameAscending);
```

Comment:
When committing to this repo, please also rev the version in the lens.json, add a brief release note line in the lens.json, and generate the zip file and include it in the PR in the dist folder.